### ISSUE TITLE
install script doesn't work

### ISSUE THREAD

**Issue by migs35323** [k3s-io/k3s#6033]
Can't seem to install with the script on ARM64, it tries to download the hash and exits with no error
tried to give flags for stable and latest

**Comment by nicklasfrahm** [comment]
Facing the same issue. It seems like the install script was changed in a way that **breaks the resolution of the release channel**. It attempts to download the hash from `https://github.com/k3s-io/k3s/releases/download/stable/sha256sum-arm64.txt`. When opening that in a browser, I get a `404 - Not Found`. Upon checking older logs, it used to download the hash from `https://github.com/k3s-io/k3s/releases/download/v1.24.3+k3s1/sha256sum-arm64.txt`.

TLDR;

```diff
+ https://github.com/k3s-io/k3s/releases/download/stable/sha256sum-arm64.txt
- https://github.com/k3s-io/k3s/releases/download/v1.24.3+k3s1/sha256sum-arm64.txt
```

**Comment by nicklasfrahm** [comment]
Upon further investigation the service hosting the lists for the different release channels appears to be down `https://update.k3s.io/v1-release/channels` returning a `503 - Service Temporarily Unavailable`.

**Comment by zanyraspi** [comment]
Facing same issue in Raspberry Pi 4. Is there a workaround?

**Comment by nicklasfrahm** [comment]
@zanyraspi As a temporary workaround you may specify a specific version of `k3s` like so:
```
curl -sfL https://get.k3s.io | INSTALL_K3S_VERSION=v1.24.4+k3s1 sh -
```

**Comment by zanyraspi** [comment]
@nicklasfrahm , thank you.

**Comment by alexfricker** [comment]
> @zanyraspi As a temporary workaround you may specify a specific version of `k3s` like so:
> 
> ```
> curl -sfL https://get.k3s.io | INSTALL_K3S_VERSION=v1.24.4+k3s1 sh -
> ```

I think you can also specify `ARCH` in the case of raspberry pi, but it may not be necessary.

**Comment by idcrook** [comment]
Also can manually install desired `k3s-ARCH` released binary and place in `/usr/local/bin/k3s` 

Then set `INSTALL_K3S_SKIP_DOWNLOAD=true`

```
$ chmod 755 k3s-arm64
$ sudo chown root:root k3s-arm64
$ sudo mv k3s-arm64 /usr/local/bin/k3s
$ ls -latr !$
ls -latr /usr/local/bin/k3s
-rwxr-xr-x 1 root root 56557568 Aug 24 22:33 /usr/local/bin/k3s
$ curl -sfL https://get.k3s.io | INSTALL_K3S_SKIP_DOWNLOAD=true ... sh -
```

**Comment by briandowns** [comment]
The channel server that provides functionality for the install script was experiencing an issue. The issue has been resolved and the install script will now behave as expected.
